src/components/transport_manager/src/transport_manager_impl.cc

```diff
@@ -856,6 +856,12 @@ TransportManagerImpl::ConnectionInternal* TransportManagerImpl::GetConnection(
   return NULL;
 }
 
+bool TransportManagerImpl::IsApplicationPresentInConnections(
+    const ConnectionUID id) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  return (nullptr != GetConnection(id));
+}
```

Comment:
@ydementieiev Is it really necessary to extend TransportManager interface? Seems the implementation of `IsApplicationPresentInConnections` is trivial and the simple check could be used instead.

Reply:
@ydementieiev I see. There's no `GetConnection` in `TransportManager`.